docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,189 @@
+
+Once you decide on a provider (or providers), if they offer instances in multiple data centers, it makes sense to spread the nodes geographically since that provides a better resilience and stability against outages.
+
+### Number of Validators
+
+Number of validators on a subnet is a crucial decision you need to make. For stability and decentralization, you should strive to have as many validators as possible.
+
+For stability reasons our recommendation is to have **at least** 5 full validators on your Subnet. If you have less than 5 validators your Subnet liveness will be at risk whenever a single validator goes offline, and if you have less than 4 even one offline node will halt your Subnet.
+
+You should be aware that 5 is the minimum we recommend. But, from a decentralization standpoint having more validators is always better as it increases the stability of your Subnet and makes it more resilient to both technical failures and adversarial action. In a nutshell: run as many Subnet validators as you can.
+
+Considering that at times you will have to take nodes offline, for routine maintenance (at least for node upgrades which happen with some regularity) or unscheduled outages and failures you need to be able to routinely handle at least one node being offline without your Subnet performance degrading.
+
+### Node bootstrap
+
+Once you set up the node instances and install AvalancheGo clients on them, nodes will need to boostrap (sync with the network). This is a lengthy process, as the nodes need to catch up and replay all the network activity since the genesis up to the present moment. Full bootstrap on a node can take more than a week, but there are ways to shorten that process, depending on your circumstances.
+
+#### State sync
+
+If the nodes you will be running as validators don't need to have the full transaction history, then you can use [state sync](../nodes/maintain/chain-config-flags.md#state-sync-enabled-boolean). With this flag enabled, instead of replaying the whole history to get to the current state, nodes simply download only the current state from other network peers, shortening the bootstrap process from multiple days to a couple of hours. If the nodes will be used for subnet validation exclusively, you can use the state sync without any issues. Currently, state sync is only available for the C-Chain, but since the bulk of the transactions on the platform happen there it still has a significant impact on the speed of bootstrapping.
+
+#### Database copy
+
+Good way to cut down on bootstrap times on multiple nodes is database copy. Database is identical across nodes, and as such can safely be copied from one node to another. Just make sure to that the node is not running during the copy process, as that can result in a corrupted database. Database copy procedure is explained in detail [here](../nodes/maintain/node-backup-and-restore.md#database).
```

Comment:
```suggestion
Good way to cut down on bootstrap times on multiple nodes is database copy. Database is identical across nodes, and as such can safely be copied from one node to another. Just make sure to that the node is not running during the copy process, as that can result in a corrupted database. Database copy procedure is explained in detail [here](../nodes/maintain/node-backup-and-restore.md#database).

Please make sure you don't reuse any node's node-id by accident, especially don't restore another node's ID, see [here](../nodes/maintain/node-backup-and-restore.md#nodeid) for details. Each node must has its own unique node id, otherwise, the nodes sharing the same id will not behave correctly, which will impact your validator's uptime, thus staking rewards, and the stability of your Subnet.
```

Reply:
Yep, grea suggestion.